guide/english/ruby/common-array-methods/index.md

```diff
@@ -160,22 +162,20 @@ array.flatten
 ```
 
 #### .join
-The .join method returns a string of all the elements of the array separated by a separator parameter. If the separator parameter is nil, the method uses an empty string as a separator between strings.
+The .join method returns a string of all the elements of the array separated by a separator parameter. If the separator parameter is nil, the method uses an empty string as a separator between strings:
 
 ``` ruby
 array.join
-=> "1234"
+=> "01234"
 array.join("*")
-=> "1*2*3*4"
+=> "0*1*2*3*4"
 ```
 
 #### .each
-The .each method iterates over each element of the array, allowing you to perform actions on them.
+The .each method iterates over each element of the array, allowing you to perform actions on them:
```

Comment:
I know this isn't related to your change but since you are modifying, this then it would be much better for the PR if you consider the following suggestion.

I suggest using pre-formatted text (similar to the html `<code>` tags) here to describe the `.each` method, you can do this by surrounding the expression by back-ticks  ` `` `